monkey/common/utils/exploit_enum.py

```diff
@@ -0,0 +1,11 @@
+class ExploitType:
```

Comment:
Again, sounds classic for an enum type, did you try it?

Reply:
When I googled I found that there is no enum for python 2.7, so I tried to come up pythonic way to do it (aenum seemed like an overkill). Only now I remembered that I've seen enum implemented  somewhere already. I'll refactor accordingly.

Reply:
https://pypi.org/project/enum34/